### ISSUE TITLE
Discussion: Start a discussion forum?

### ISSUE THREAD

**Issue by docwilmot** [backdrop-ops/backdropcms.org#18]
I was interested in the concept of Backdrop from the start, but I am not sure how to contribute. GitHub seems (to me) to be a place for code, not community, and I notice that most open source projects pride themselves on a 'thriving community'. Is it too early to start Backdrop Forums?

Options for places to have a forum

<table style="width:664px">
    <thead>
        <tr>
            <th>Option</th>
            <th>Good</th>
            <th>Bad</th>
        </tr>
    </thead>
    <tbody>
                <tr>
            <td>Reddit</td>
            <td>
               <ul>
                <li>Free</li>
                <li><a href="http://www.reddit.com/r/backdrop">Immediately available</a></li>
                <li>Will happen anyway</li>
                <li>Great for non-technical people</li>
                <li>Can customize look and feel</li>
                   </ul>
            </td>
            <td>
              <ul>
                <li>Requires a Reddit login</li>
                <li>No SSO w/ Github</li>
            </ul>
        </tr>
                <tr>
            <td>Facebook group</td>
            <td>
               <ul>
                <li>Free</li>
                <li><a href="https://www.facebook.com/backdropcms">Immediately available</a></li>
                <li>Will happen anyway</li>
                <li>Great for non-technical people</li>
                <li>Everyone already has facebook (admit it, you do)</li>
                   </ul>
            </td>
            <td>
              <ul>
                <li>Requires a Facebook login</li>
                <li>No SSO w/ Github</li>
            </ul>
        </tr>
        <tr>
            <td>Zoho Forums</td>
            <td>
               <ul>
                <li>Free</li>
                <li><a href="https://discussions.zoho.com/backdrop">Immediately available</a></li>
                <li>SSO w/ Github</li>
                <li>JQuery chose it</li>
                   </ul>
            </td>
            <td>
              <ul>
                <li>Only one forum (free version)</li>
            </ul>
        </tr>
        <tr>
            <td>Stack Exchange</td>
            <td>
               <ul>
                <li>Free</li>
                <li>Immediately available</li>
                <li>Will happen anyway</li>
                <li>SSO w/ lots of things</li>
                   </ul>
            </td>
            <td>
              <ul>
                <li>It's not a discussion forum.</li>
                <li>Credit needs to be earned</li>
                <li>No SSO w/ Github</li>
            </ul>
        </tr>
        <tr>
            <td>Google groups</td>
            <td>
              <ul>
                <li>Free</li>
                <li> <a href="https://groups.google.com/forum/#!forum/backdrop-builders">Immediately available</a> </li>
              </ul>
            </td>
            <td>
              <ul>
                <li>Requires a google account</li>
                <li>Does not let you choose a new persona</li>
                <li>No SSO w/ Github</li>
            </ul>
            </td>
        </tr>
        <tr>
            <td>Backdrop<br/>
                         Forum module</td>
            <td>
            <ul>
                <li>Module Already ported</li>
                <li>SSO w/ Github</li>
            </ul>
            </td>
            <td>
                <ul>
                <li>Requires a  Backdrop login</li>
                <li>Needs to be built (dev resources)</li>
                <li>Needs to be maintained (dev resources)</li>
                <li>Needs to be hosted (cost)</li>
                <li>proven unsuccessful on d.o</li>
                <li>Old technology</li>
            </ul>
             </td>
        </tr>
        <tr>
            <td>Backdrop<br/>
                        Views/Content type</td>
            <td>
                <ul>
                <li>Config-ready</li>
                <li>SSO w/ Github</li>
                </ul>
            </td>
            <td>
                <ul>
                <li>Requires a Backdrop login</li>
                <li>Needs to be built (dev resources)</li>
                                <li>Needs to be maintained (dev resources)</li>
                                <li>Needs to be hosted (cost)</li>
                <li>proven unsuccessful on d.o</li>
            </ul>
                        </td>
        </tr>
        <tr>
            <td>Discourse</td>
            <td>
              <ul>
                <li>Immediately available</li>
                <li>Open Source</li>
                <li>Ubuntu chose it</li>
              </ul>
            </td>
            <td>
              <ul>
                <li>Requires a Discourse login</li>
                <li>Needs to be built (dev resources)</li>
                                <li>Needs to be maintained (dev resources)</li>
                                <li>Needs to be hosted (cost)</li>
                <li>OR Would have to buy (cost)</li>
                <li>Not Backdrop</li>
                <li>No SSO w/ Github</li>
              </ul>
            </td>
        </tr>
    </tbody>
</table>

**Comment by BWPanda** [comment]
@jenlampton, @quicksketch: I think we need to set something up for the launch of BD so there's somewhere people can go for support when they start using it.

An issue queue-only GH repo. seems the best temporary solution at this point...

**Comment by quicksketch** [comment]
We already have the core issue queue (https://github.com/backdrop/backdrop/issues) with the "question" tag, that we've already been using for that purpose. Long-term I still think a better solution is necessary, but wouldn't that work instead of creating another repository?

**Comment by jenlampton** [comment]
I often forget to check on our other repositories :(

**Comment by BWPanda** [comment]
> ...wouldn't that work instead of creating another repository?

I guess so :smile: I thought there was originally a reason for not wanting to fill-up the developer-based issue queue with support questions, but considering we're just wanting a temporary solution for now, and considering @jenlampton's comment about forgetting to check other repos (which I also forget to do sometimes), using 'question' tags in the main issue queue seems like a good compromise.

**Comment by klonos** [comment]
> ...using 'question' tags in the main issue queue seems like a good compromise.

Until we build a dedicated forum on Backdrop CMS to 1) show off that it can be done and 2) port any contrib modules that are required in order to build such a Q&A site.

**Comment by docwilmot** [comment]
> Until we build a dedicated forum on Backdrop CMS to 1) show off that it can be done and 2) port any contrib modules that are required in order to build such a Q&A site.

Agreed. [Of all the modules removed from core](https://api.backdropcms.org/node/28538) I think forum.module would very likely affect our commitment to a pain-free D7 to Backdrop upgrade. I'm pretty sure a significant percentage of the Backdrop audience (small to medium sites) would have forum.module enabled.
Even if its not core, it would be best to port it. It's a bit beyond me (just tried, and I have the headache to prove it).

**Comment by quicksketch** [comment]
Yep, I'd love to see forum module restored (with an upgrade path) in contrib. There's a lot of area for improvement of course, but it would be great to have a native forum solution working in Backdrop.

**Comment by docwilmot** [comment]
Done porting forum module although haven't looked at tests yet.

**Comment by cellear** [comment]
Hi guys.  In order to quickly have a place where non-expert users can get their bearings, I've set up a Google Group called "Backdrop Builders." It is intended specifically to address the needs of people that just want to _use_ Backdrop to create websites, rather than try to build or extend it.

https://groups.google.com/forum/#!forum/backdrop-builders

I have a lot of experience managing online communities, so I'll keep it moving, but I welcome any help or support (or just subscribers) that want to join in.

**Comment by docwilmot** [comment]
This morning a new user (an experienced Drupalist it sounds like) said:
> Gitbub is usability nightmare for me. No idea how to find the project queues

Which got me to thinking we really need to streamline/centralize information sources. There are just too many GitHub locations, all without a unified navigation to get you from one to the other.

So I think adding another unrelated source to it in the form of a Google group is probably not the best way to go. 

* We need a user forum on Backdrop.org
* We need a list (pulled from GH RSS perhaps) of contribs. Alternatively, and maybe better, I think in lieu of project.module it may be an idea to have a node type called project, which new contrib authors fill out to add a new contrib. That way we can track contrib types (module/theme/layout) and be able to generate lists with links that direct to the issue queues.
* We could also have similar RSS lists for the multiple GH issue queues, with links.

I think the group is growing and we should act to have infrastructure for support in place now so we don't end up with multiple locations to manage.

**Comment by jenlampton** [comment]
I disagree about using forum module (and don't think it has a place in Backdrop contrib either).  The reason we removed it from core is because we don't need a module for forums now that we have config.  A forum should be a content type (config) and some views (config).  After we get a features-module for Backdrop, there could be a forum feature that would bundle together the config that is needed - but the forum module as removed from Drupal 7 is how these problems were solved in the past, not how they should be solved in the future (and present?). 

That said, it would be nice to have someplace for people to go to chat that's not developer focused like Github. I'm not sure backdropcms.org is the place to do it though,  because people won't be incentivized to create user accounts there.

I'm not sure there's a clear winner here...

**Comment by cellear** [comment]
I think Drupal suffers greatly from its attempt to centralize user-contributed information exchange.  The Drupal handbook and forums are both of very limited value for helping new people successfully build a Drupal 7 site.  I tell people new to Drupal that they need to buy two books -- O'Reilly's "Using Drupal" and "Pro Drupal Development" -- in order to get on top of things.  For free online resources, I point them to "The Weekly Drop" (http://www.theweeklydrop.com/) -- Bob has done more to distribute actual usable information on how to get things done in Drupal than any group I know.

So I thought it would be useful to provide a place for people who don't have and will never have a Github account to find a smattering of information about Backdrop.  A place where they could browse and possibly find the answer they're looking for, and where they won't be afraid to ask a question if they don't find it.  This shouldn't stop us from building up our own, centrally-managed and curated docs.  I would love to see us be as approachable, or more, than Wordpress and Node.JS are.  

Don't worry about managing the Google group; I'll take care of it.  (Or it will die and be quickly forgotten.)

**Comment by docwilmot** [comment]
> That said, it would be nice to have someplace for people to go to chat that's not developer focused like GitHub

Agreed, and soon.
> A forum should be a content type (config) and some views (config)

Agree and disagree, I considered that before trying to port forum.module but the absence of a features-like glue was the main issue against that. And I thought that this would be better than nothing. And besides forum module is sort of a glue module for forum content types ;)
> don't think it has a place in Backdrop contrib either

Disagree there. I think a user should be free to choose whatever forum he likes. If he/she prefers Forum over Discourse or vBulletin or SXC or a Views/Content mashup, or whatever, so be it. 

> The Drupal handbook and forums are both of very limited value for helping new people successfully build a Drupal 7 site

I'm pretty sure those two are where I learnt everything about websites!

**Comment by quicksketch** [comment]
I'll also add that I'm happy to see @docwilmot's efforts on porting Forum module, so there's an upgrade path available for users of Forum for D7. Perhaps long-term it will turn into more of a Views + Config module, now that we have those tools. But for now I think it's great that the contrib version can maintain continuity until that becomes reality.

**Comment by cellear** [comment]
"I'm pretty sure those two are where I learnt everything about websites!"

I think they used to be more suitable than they are now.  How long ago were you using them?  I got into Drupal near the end of D6, and much of what I found in the Handbook and Forums was already hopelessly outdated.  Answers often talked about implementing hooks in "your module" -- but a CMS shouldn't assume you're writing a module.  If you limit your audience to those those people that feel comfortable writing PHP code, you'll rule out 95% of your potential users before you start.  With Drupal 7, you can actually build very sophisticated sites without writing code, if you can find a suitable theme (usually the hardest part.)  But from the documentation you would probably not guess it!

I learned PHP back in the 90s, but turned to Drupal largely hoping to avoid having to maintain my own source code for every website I built.  The handbook slowed me down more than it ever helped me.

**Comment by chrisherberte** [comment]
Hi all, the Backdrop website to me looks terribly unwelcoming for newcomers and especially so for non-technical users. Maybe to add some clarity It's been said already GH in not the place for discussion beyond contrib issues and support.

Maybe an externally hosted community solution is the answer short term and longer term? This is essentially the question that needs answering to move forwards on this topic.

Centralising project pages with some level of GH integration is absolutely necessary and IMHO should be a completely separate to community forums.

**Comment by BWPanda** [comment]
> I've set up a Google Group called "Backdrop Builders."

@cellear I'm not sure how Google Groups work, but I'd advise against making it so that users "must be signed in and a member of [the] group to view and participate in it" (the message I see when I try to access it).

If there's an option to allow 'anonymous' users to access the group, I'd recommend setting that. If not, maybe Google Groups isn't the best (we decided against StackExchange for similar user-unfriendly reasons).

**Comment by chrisherberte** [comment]
Google Groups is definitely not the answer, John Resig demonstrated that the hard way with jQuery support community.

**Comment by cellear** [comment]
Oops, I had it set to only be readable by members.  That's obviously not very helpful.  I've changed it to be viewable to non-logged in users, so that at least is now fixed.

What did JQuery decide to go with?

**Comment by chrisherberte** [comment]
@cellear Zoho sponsor jQuery and provide their discussion product IIRC.

**Comment by cellear** [comment]
A little research has shown that Jquery switched to Zoho Forums (https://discussions.zoho.com/home) for reasons listed here: http://jquery14.com/day-07/new-jquery-forum.

However, I note that they were able to use Google Groups for four years before they switched, amassing over 50,000 threads.  So it worked pretty well for quite a while.  The JQuery group was using Google Groups to support everything -- including all of the communication among developers.  They shut it down 5 years ago; they probably wouldn't have started it on GG if StackExchange had existed at the time.

I'm not sure that we're talking about the same audience.  I set up the Google group specifically to try to help non-programmers be successful with Backdrop.  Most of the questions I foresee ending up there probably won't be allowed on StackExchange anyway!  I'm certainly not suggesting that my little group should supplant anything that already exists, I just thought it would make sense to have a place where users who were not developing the system could talk about how to use Backdrop as it exists, as opposed to how to fix it or improve the system itself.  (Note the title: "Backdrop Builders," as opposed to "developers" or "themers.")

**Comment by jenlampton** [comment]
So, should we give [Zoho Forums](https://discussions.zoho.com/home) a try?

**Comment by BWPanda** [comment]
> I also just got an invite to Gittter

Me too; was wondering why. It seems to be developer-centric...

**Comment by BWPanda** [comment]
> I've changed it to be viewable to non-logged in users...

Great, thanks!

**Comment by docwilmot** [comment]
Whats wrong with discourse?

**Comment by jenlampton** [comment]
With Discourse the software is free, but we'd either need to build/support it, or pay for a service that would do the same.  If we're going to build/support something it should at least be made of Backdrop, since that's what we'll be experts at. Ideally we'd be able to leverage something that already exists so we don't have any additional infrastructure that needs to be maintained (or, at least not until we have the manpower/dollars/time to do so).

**Comment by cellear** [comment]
Discourse is a hosted app based on RoR, Ember, and PostgreSQL; these are perfectly fine environments, but it would seem to be a rather extreme case of _not_ "eating our own dog food."  If we're going to host our own forums, we should probably just build it in Backdrop.  But I think there's a lot to recommend a hosted service.

**Comment by chrisherberte** [comment]
How about approaching Discourse for a "free for non profit" account?

**Comment by jenlampton** [comment]
@chrisherberte we could try, but we don't officially have a non-profit status yet. We are still waiting to for our application to be considered by the [Software Freedom Conservancy](http://sfconservancy.org).  Usually they are pretty strict about you actually being a not-for-profit before they give you something for free.

**Comment by docwilmot** [comment]
Let's all now pick one:

<table style="width:664px">
	<thead>
		<tr>
			<th>Option</th>
			<th>Good</th>
			<th>Bad</th>
		</tr>
	</thead>
	<tbody>
		<tr>
			<td>Backdrop on Forum module</td>
			<td>
			<ul>
				<li>Already ported</li>
				<li>Can be converted to Views/Content type/config later</li>
			</ul>
			</td>
			<td>Old technology</td>
		</tr>
		<tr>
			<td>Backdrop on Views/Content type</td>
			<td>Would need building</td>
			<td>Config-ready</td>
		</tr>
		<tr>
			<td>Discourse</td>
			<td>Immediate if we could get it</td>
			<td>
			<ul>
				<li>Would have to build or buy</li>
				<li>Outside Backdrop, requires another login</li>
			</ul>
			</td>
		</tr>
		<tr>
			<td>Google groups</td>
			<td>Immediate (done already)</td>
			<td>
			<ul>
				<li>Outside Backdrop, requires another login</li>
				<li>Uses personal G+ info for account rather than a separate username/personna</li>
			</ul>
			</td>
		</tr>
		<tr>
			<td>Zoho Forums</td>
			<td>Immediately available</td>
			<td>One forum, no subforums in free version</td>
		</tr>
	</tbody>
</table>

**Comment by cellear** [comment]
@docwilmot : Would you be willing to elaborate on the kinds of discussions you had in mind when you said "Is it too early to start Backdrop Forums?"  in the OP?  There's a wide range of possible answers, and I'm not convinced we're all talking about the same things.  Specifically, could you touch on the kinds of users you see being involved and benefitting?  

I'm surprised to see the GG I set up in the list, but perhaps I shouldn't be.   I wasn't picturing backdrop-builders being an "official" Backdrop solution at all; it was just a little thing I set up.  For better or worse, I was approaching it as an outsider, not an insider.  I thought there should be a place where people could ask about things that all "Drupal people" know without feeling foolish.  For example, "What is a themer?"  (I answer that one all the time; most of my clients have no idea what the difference is between a themer, a developer, a site builder, and a content administrator.)

**Comment by docwilmot** [comment]
> I thought there should be a place where people could ask about things that all "Drupal people" know without feeling foolish

Exactly that, except "**Backdrop** people" :)

And I absolutely was approaching this as an outsider back then, and still am; BD was new and interesting, it was (to be) a community-based project, I thought, as is Drupal, WP, Joomla etc, and I wondered where people in the community should hang out and ask questions without bothering the coders, and where people who may know a little bit more but not enough to write the big code could hang out and share what they know.

And Google groups is an absolutely reasonable means to this end. People build communities there, and thats what we need. My only gripe was that I think BD fora should be built by BD. I could be wrong about that though.

**Comment by chrisherberte** [comment]
> Backdrop on Views/Content type
I pick Option 2 in docwilmot's table. It would provide a single forum solution immediately. Keep it simple.

**Comment by jenlampton** [comment]
I don't like ideas 1 or 2 because they both involve making people log in to backdropcms.org. That's something developers will be doing to post official pages for their modules, but not something the people @cellear and @docwilmot are talking about are likely to do.

I also don't like the idea of pouring precious dev resources into something for Backdrop that has proven to be unsuccessful for the Drupal project.

I think it would be better if we used something that already existed first.  When we find something that works for us, then we can talk about moving it to our site, and building / maintaining it ourselves.

**Comment by BWPanda** [comment]
Not sure about the short-term, but in the long-term I think we should go with option 2: **'Backdrop
Views/Content type'** forums. If we can make something really cool that works well, we could even package it up and release it similar to how WordPress did: https://bbpress.org/

To avoid the 'logging into backdropcms.org' problem, we could make it its own site; something like support.backdropcms.org...

**Comment by chrisherberte** [comment]
yes, or community.backdropcms.org The contributed projects pages could live there too.

**Comment by jenlampton** [comment]
All the separate sites will eventually use bakery module (needs to be ported) and sign a person into all sites at once (www, api, support, etc) the problem is that people don't like creating MORE accounts.  making them create 3 more is worse than 1 more ;)

**Comment by jenlampton** [comment]
A facebook group!

**Comment by chrisherberte** [comment]
You should be banned for that comment.

**Comment by klonos** [comment]
> You should be banned for that comment.

++

**Comment by jenlampton** [comment]
LOL <3 you guys :)

**Comment by klonos** [comment]
...no seriously. :p

**Comment by BWPanda** [comment]
So if we want our clients to be able to participate in a support community, but don't want them to have to register a new account, how will that work with something like Zoho, Facebook or Google? Are we assuming those places have the ability to login using other services (e.g. login to Zoho using Facebook) or that clients already have an account there?

**Comment by jenlampton** [comment]
It depends on the service we choose, I suppose. I'd be fine with either.

**Comment by cellear** [comment]
Let's see if one of us can find a client willing to pay for a custom Backdrop-based forums solution and give permission to donate it back!

**Comment by sirkitree** [comment]
Jen added a table to the original post above. I've added a couple more pros to Reddit as a forum.

I was pretty skeptical of Reddit at first when I was introduced to the VR community, but it's proven a really great forum. Moderation, stylization, posting guidelines, among others. Hope to see ya'll there!

**Comment by jwalling** [comment]
In case it wasn't mentioned, it would be good to have a shareable event calendar connected to the forum.

**Comment by sirkitree** [comment]
Connected how @jwalling? We can put a link to it from the subreddit, though I'm not sure where that link is at this point.

**Comment by jwalling** [comment]
@sirkitree
Connected: Easy to link w/o need to use a separate password.

**Comment by jwalling** [comment]
Ideally, it should be easy to post to the forum and calendar using one login, as in Drupal Groups.

**Comment by docwilmot** [comment]
> proven unsuccessful on d.o

This is the comment for on-site forums. Just curious why you think it is unsuccessful? Hadnt realized people thought it was unsuccessful.

**Comment by mikl** [comment]
As far as I'm concerned, the rampant NIH-ism is one of the worst features of the Drupal Community. We have Drupal.org with a half-assed issue tracker, half-assed continuous integration service, half-assed forums, half-assed group-platform, half-assed documentation wiki, etc.

All those things work, of course, it's just that they are, in most aspects, worse than other available solutions. And they suck down an immense amount of volunteer work to maintain – work that could otherwise have been funnelled into making Drupal itself better.

As for dog-fooding, I've never seen any convincing evidence that this actually significantly helped drive adoption of Drupal. What drove adoption of Drupal when that took off was that big-name sites were using it as CMS – whitehouse.gov was a huge turning point.

So I think the rational approach would be for the Backdrop community to focus on making a great CMS, instead of trying to build yet another forum software.

I really think Discourse is the best forum software out there. Designed towards getting great conversations going and getting out your way. It's no accident that it's been picked up by big open source projects like Ubuntu (http://discourse.ubuntu.com/ )

I already host a few Discourse instances for smaller projects, and the Backdrop community would like to try it, I can host an instance for you as well, free of charge as long as you don't start having thousands of concurrent users.

**Comment by chrisherberte** [comment]
@miki this reinforces my thinking that the Backdrop CMS community should be handled by "the community". There is sure to be plenty of discussion about the logistics of your proposal, I think it's possibly a step in the right direction.

**Comment by docwilmot** [comment]
OK I've been to the Backdrop Reddit started by @sirkitree and it looks good. 
BUT I wanted to bring up one, slightly embarassingly simplistic personal thought about Reddit because I am in no way knowlegable about that site:
Reddit's reputation to my mind doesnt seem fantastic at this time. I hear Reddit and I think hackers and Jennifer Lawrence. If I am overreacting, then Reddit otherwise sounds like a good idea. If I'm not does it harm Backdrop to be there?
Am I overreacting?

**Comment by sirkitree** [comment]
Reddit is pretty prolific, and if you follow the crap that makes it to the front page, it looks terrible. Subreddits are their own communities though, and can be good or bad, depending on the people that moderate and the people who frequent it. The overall reputation of Reddit should not be a limiting factor IMHO, but I can certainly see where you're coming from.

**Comment by mikl** [comment]
Yeah, Reddit is a bit like a microcosm of the internet. There's a lot of really terrible stuff there, but you don't have to go to those subreddits, if you don't want. I've been a Redditor for ages, and I rarely see anything objectionable.

**Comment by jenlampton** [comment]
@mikl I think Discourse is off the table because it has all the same down-sides as eating our own dogfood (support, maintenance, hosting) plus, it has the additional disadvantage of also being technologies that we are not experts in.  Though your offer to let us take it for a spin on your servers is quite generous, we will be in a pickle the day we do start to have thousands of concurrent users.

Discourse may in fact be the best forum software, but I don't think it's the best choice for this community. 

I took the liberty of organizing the list in the top post in (roughly) best-to-worst, in terms of pros and cons. The top four or five items all have links to where we can try things.  

Let's start by giving Reddit a shot, but we can also try Zoho Forums, Facebook (agad!) Google Groups, and Stack Exchange.  Let's see what works and what doesn't work and see if we need to give anything else a try, or if one of those becomes a clear winner.

**Comment by cellear** [comment]
Since we have Facebook Groups on the list, it seems to make sense to mention Google Plus.  Is https://plus.google.com/+BackdropcmsOrg/posts group a discussion forum, or is it reserved for PMC announcements?  (You could make a case for LinkedIn groups as well, as the Backdrop job market gathers steam.)

**Comment by jenlampton** [comment]
I thought google plus was dead?  Officially, we just use the hangouts - no announcements or anything there.

**Comment by cellear** [comment]
G+ certainly didn't kill off Facebook, as some were predicting.  (Neither did Ello!) But it has some pretty thriving discussions on various vertical topics that people are into. Or so I've heard :)

**Comment by cellear** [comment]
The LinkedIn group is apparently moderated.  Are you the owner, @jenlampton?

**Comment by jenlampton** [comment]
I think I am.  Checking group settings says... "Any member of LinkedIn may join this group without requiring approval from a manager."

**Comment by docwilmot** [comment]
So Reddit it is then. 
We should put a link and an announcement on Backdrop.org.
And change the logo there from the dragon to the Backdrop logo?

**Comment by jenlampton** [comment]
The header has the backdrop logo. I'll let you guys decide if other logos need to be changed :)

**Comment by sirkitree** [comment]
@docwilmot what changes would you like to see? Would you mind taking the conversation there, posting a thread on changes you'd like to see. Totally open to changes but we should probably keep the conversation about changing the subreddit there.

**Comment by jenlampton** [comment]
@docwilmot brought this back to our attention over in https://github.com/backdrop-contrib/webform/issues/2#issuecomment-128151760 

I think it may be time to revisit this discussion.  I also like the idea of people being able to "join something" even if they do nothing else there. 

First, let's decide on metrics of success, so we can tell if all the extra time/effort/money that's not going into Backdrop the software is benefiting Backdrop the community in a significant way. Then, let's give a few things a shot! :)

**Comment by kreynen** [comment]
As much as I advocate open source and standards, it's hard to argue with how much more people enjoy Slack over IRC.  Code like https://www.drupal.org/project/slack_invite opens up what is normally an invite only process to something anyone can join.  WordPress.org moved their core conversations to Slack.  As far as know, they never looked back.

**Comment by jenlampton** [comment]
This issue can continue to be used to discuss whether we *should* try to build our own discussion forum, but we also need to start thinking about how we'd do it and what it would look like, to better inform that decision. I've started two additional issues related to building a discussion forum:
* https://github.com/backdrop-ops/backdropcms.org/issues/249 (Content)
* https://github.com/backdrop-ops/backdropcms.org/issues/250 (Implementation)

**Comment by jenlampton** [comment]
Since it sounds like we have decided to build our own forum, I am going to close this issue. Thanks everyone for participating! *eats our own dogfood*
